.github/workflows/gmc-e2e.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+name: gmc e2e test
```

Comment:
Please use golang not gmc in your test workflow. We hope this workflow is generic to all golang project.